### PR TITLE
More select mode options for users

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: openslides/users/static/js/users/site.js [hunk 1]
```diff
@@ -609,60 +613,58 @@ angular.module('OpenSlidesApp.users.site', [
...
+            });
         };
         // add group for selected users
         $scope.addGroupMultiple = function (group) {
-            angular.forEach($scope.users, function (user) {
-                if (user.selected) {
-                    user.groups_id.push(group);
-                    User.save(user);
-                }
+            if (!group) return;
```
Comment (normanjaeckel) [inline openslides/users/static/js/users/site.js]: Please add `{ }` and please use positive condition check: if group then call selectModeAction ...

File: openslides/users/static/js/users/site.js [hunk 2]
```diff
@@ -609,60 +613,58 @@ angular.module('OpenSlidesApp.users.site', [
...
         // remove group for selected users
         $scope.removeGroupMultiple = function (group) {
-            angular.forEach($scope.users, function (user) {
-                if (user.selected) {
-                    var groupIndex = user.groups_id.indexOf(parseInt(group));
-                    if (groupIndex > -1) {
-                        user.groups_id.splice(groupIndex, 1);
-                        User.save(user);
-                    }
+            if (!group) return;
```
Comment (normanjaeckel) [inline openslides/users/static/js/users/site.js]: Here the same.

**Comment by normanjaeckel** [comment]
Also add CHANGELOG entry if reasonable.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Done. Added general note for more multiselect actions (includes also generate password etc.)

File: openslides/users/static/js/users/site.js [hunk 3]
```diff
@@ -609,60 +613,60 @@ angular.module('OpenSlidesApp.users.site', [
...
+        $scope.deleteMultiple = function () {
+            selectModeAction(function (user) {
+                $scope.delete(user);
+            });
         };
         // add group for selected users
         $scope.addGroupMultiple = function (group) {
-            angular.forEach($scope.users, function (user) {
-                if (user.selected) {
+            if (!group) {
```
Comment (normanjaeckel) [inline openslides/users/static/js/users/site.js]: Are you sure that this is `!group` instead of `group`?

Comment (FinnStutzenstein) [inline openslides/users/static/js/users/site.js]: Oh, I think I didn't run gulp, so I didn't noticed..; Now its fixed

**Pull request merged**